community/kernel/src/main/java/org/neo4j/kernel/impl/store/kvstore/ConcurrentMapState.java

```diff
@@ -73,12 +66,10 @@ public String toString()
     @Override
     public EntryUpdater<Key> updater( long version, Lock lock )
     {
-        if ( versionSequence.seen( version, META ) )
+        if ( version <= previousVersion )
         {
-            throw new IllegalStateException( "Cannot apply update with given version " + version +
-                                             " when base version is " + previousVersion );
+            return EntryUpdater.noUpdates();
```

Comment:
Our count store rotation based on last committed transaction and since we publish transaction as committed before apply transaction to count store that _can_ be the case